docs-src/dev-guide/major-components.md

```diff
@@ -0,0 +1,263 @@
+---
+id: major-components
+title: Temporal SDKs major components
+description: Temporal SDKs provide developers with a Temporal Client to communicate with a Temporal Cluster, APIs to define Workflows, APIs to configure and start Workers, APIs to customize Activity Executions, and a common library for code that’s used across your application.
+sidebar_label: Major components
+tags:
+  - temporal sdk
+  - explanation
+  - components
+---
+
+**What are the major components of Temporal SDKs?**
+
+Temporal SDKs offer developers with the following:
+
+- A Temporal Client to communicate with a Temporal Cluster
+- APIs to develop application code Workflows & Activities
+- APIs to configure and run Workers
+- A common library for code that’s used across the Client, Worker, and Workflow
+
+![Temporal SDK components create a runtime across your environment and a Temporal Cluster](/diagrams/temporal-sdk-components.svg)
+
+Let’s break down each one.
+
+### Temporal Client
+
+A Temporal Client acts as a bridge of communication between the systems outside of Temporal and your application.
+The Client performs key functions that facilitate the execution of, management of, and communication with Workflows.
+
+The following code is an example using the Go SDK.
+It showcases how to initialize a Temporal Client, create a connection to a local Temporal Cluster, and start a Workflow Execution:
+
+:::caution Do not copy and use code
+
+The following code is for example purposes only.
+For tested code samples and best practices, use your preferred language SDK's developer's guide.
+
+:::
+
+```go
+package main
+
+import (
+	"context"
+
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+  // Temporal Client setup code
+	c, err := client.NewClient(client.Options{})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+  // Prepare Workflow option and parameters
+	workflowOptions := client.StartWorkflowOptions{
+		ID:        "loan-application-1",
+		TaskQueue: "loan-application-task-queue",
+	}
+	applicantDetails := ApplicantDetails{
+		// ...
+	}
+  // Start the Workflow
+	workflowRun, err := c.ExecuteWorkflow(context.Background(), workflowOptions, "loan-application-workflow", applicantDetails)
+	if err != nil {
+		// ...
+	}
+	// ...
+}
+```
+
+Developers can then use the Client as the main entry point for interacting with the application through Temporal.
+From here, developers can now start or Signal Workflows, query the Workflow state etc.
+We can see in the example above how the developer has used the provided start method to start the Workflow upon the running of the function.
```

Comment:
> how the developer has used the provided start method

It was unclear to me what you mean by "provided start method." Are you referring to the entire program (which I assume would be named `start/main.go`), the `client.ExecuteWorkflow` call, or something else? Regardless, I would avoid the word "method" since that may be interpreted in the object oriented way. Maybe "has used provided means of starting the Workflow" would be more clear.

Reply:
> to start ~the~ Workflow ~upon the running of the function~.

to start **a** Workflow.